tools/ALARAJOYWrapper/njoy_tools.py

```diff
@@ -245,14 +293,22 @@ def run_njoy(element, A, matb):
     # If the run is successful, log out the output
     # and make a copy of the file as a .GENDF file
     if not result.stderr:
-        gendf_dir = 'gendf_files'
-        if not os.path.isdir(f'{dir}/{gendf_dir}'):
-            os.mkdir(f'{dir}/{gendf_dir}')
-        gendf_path = f'{dir}/{gendf_dir}/tendl_2017_{element}{str(A).zfill(3)}.gendf'
-        Path('tape31').rename(Path(gendf_path))
-        ensure_gendf_markers(gendf_path, matb)
+        for filetype in save_dict.values():
+            save_path = f'{dir}/{
+                        filetype['Specific Dir']
+                        }/tendl_2017_{element}{str(A).zfill(3)}'
+            
+            if not os.path.isdir(
+                os.path.join(dir, filetype['Specific Dir'])
+                ):
+                    os.mkdir(os.path.join(dir, filetype['Specific Dir']))
```

Comment:
`Path.mkdir()` allows you to set `exist_ok=True` so you can just make the directory even if it exists already.